annotation/src/main/java/com/android/designcompose/annotation/Builder.kt

```diff
@@ -50,15 +50,39 @@ annotation class DesignComponent(
  *
  * @param node the name of the Figma node
  */
+// TODO adding AnnotationTarget.PROPERTY to the list in @Target doesn't work. The annotation will
```

Comment:
(Oops, I commented in the wrong place!). Please add the link to the KSP ticket in these comments.

Reply:
Created a KSP bug since it didn't exist and reference it here.